Dockerfile

```diff
@@ -0,0 +1,88 @@
+# syntax=docker/dockerfile:1
+
+ARG CUDA_VER=11.8.0
+ARG PYTHON_VER=3.10
+ARG LINUX_VER=ubuntu22.04
+
+ARG RAPIDS_VER=23.08
+ARG DASK_SQL_VER=2023.6.0
+
+ARG BASE_FROM_IMAGE=rapidsai/mambaforge-cuda
```

Comment:
does this need to be an `ARG`?

as far as I can tell it's only used in one spot and its value will never change.

Reply:
No, it doesn't. This was from when I was concurrently updating `mambaforge-cuda` and `docker` to support the `conda` group. Just made it easier to swap the parent image to PR tag on `mambaforge-cuda`.

But I'll remove it.